src/components/atoms/InternalAccordion/useAccordionContext.tsx

```diff
@@ -0,0 +1,15 @@
+import { useContext } from "react";
```

Comment:
This file should be .ts  not .tsx .  If there is no HTML like tags then it's always .ts